igvm/settings.py

```diff
@@ -206,10 +206,9 @@
     'stopped': 80,
 }
 
-AWS_GRP_NAME = environ.get('AWS_GRP_NAME', default='adm')
 AWS_INSTANCES_OVERVIEW_URL = 'https://www.ec2instances.info/instances.json'
-AWS_INSTANCES_OVERVIEW_FILE = '/tmp/AWS_INSTANCES_OVERVIEW_FILE.json'
-AWS_INSTANCES_OVERVIEW_FILE_ETAG = '/tmp/AWS_INSTANCES_OVERVIEW_FILE.etag'
+AWS_INSTANCES_OVERVIEW_FILE = 'AWS_INSTANCES_OVERVIEW_FILE.json'
+AWS_INSTANCES_OVERVIEW_FILE_ETAG = 'AWS_INSTANCES_OVERVIEW_FILE.etag'
```

Comment:
This will still cause problems if the user wants to build multiple VMs in parallel. Please use https://docs.python.org/3/library/tempfile.html instead.